devtools/aptos-cargo-cli/src/lib.rs

```diff
@@ -27,6 +27,19 @@ const RELEVANT_FILE_PATHS_FOR_COMPILER_V2: [&str; 7] = [
     "aptos-move/move-examples",
     "third_party/move",
 ];
+const NO_MVC_BLOCK_V1_PACKAGES: [&str; 11] = [
```

Comment:
@vineethk This is the list of crates where `MVC_BLOCK_V1` is not set when testing them. Some of them will be removed in a separate PR.